### PR TITLE
feat(swarm-test): allow publishing to crates.io

### PR DESCRIPTION
**Pull request by mxinden**
## Description

Allows folks like @bajtos to use the crate, see https://github.com/filecoin-station/zinnia/pull/85.

I opted for not exposing the crate through the `libp2p` meta crate as it is a testing tool only. @thomas
let me know if you prefer me to do so.

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: swarm-test/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# 0.1.0 - unreleased
+
+- Initial release.
```
Comment (thomaseizinger) [inline swarm-test/CHANGELOG.md]:
```suggestion
- Initial release.

```

**Review by thomaseizinger — APPROVED** [review]
LGTM

No need for re-export.

**Comment by thomaseizinger** [comment]
cc @melekes

File: swarm-test/Cargo.toml
Comment (thomaseizinger) [inline swarm-test/Cargo.toml:1]: Need to set `rust-version`.

File: swarm-test/CHANGELOG.md (hunk 1 above)
Comment (mxinden) [reply swarm-test/CHANGELOG.md]: Why an additional newline? Note that there is already one newline character at the end of the final line, thus GitHub is not complaining.

**Comment by bajtos** [comment]
Awesome, thank you guys! 👏🏻

File: swarm-test/CHANGELOG.md (hunk 1 above)
Comment (thomaseizinger) [reply swarm-test/CHANGELOG.md]: Hmm, it is for me?

**Review by thomaseizinger — CHANGES_REQUESTED** [review]
`libp2p-swarm-test` is wrongly included as a production dependency in `libp2p-perf` hence that build is now failing.

File: swarm-test/CHANGELOG.md (hunk 1 above)
Comment (mxinden) [reply swarm-test/CHANGELOG.md]: Strange. Neither on Firefox nor Chromium it shows for me. Though I was wrong, there is no newline. Will push a commit.

**Review by thomaseizinger — APPROVED** [review]

**Comment by thomaseizinger** [comment]
@mergifyio refresh

**Comment by thomaseizinger** [comment]
I think before we want to release this, we also need to specify a version for each dependency and not just a path reference.

**Pull request merged**